src/Assets/BootstrapAsset.php

```diff
@@ -29,10 +30,19 @@ class BootstrapAsset extends AssetBundle
         'js/bootstrap.bundle.js',
     ];
 
-    public array $publishOptions = [
-        'only' => [
-            'css/bootstrap.css',
-            'js/bootstrap.bundle.js',
-        ],
-    ];
+    public array $publishOptions = [];
```

Comment:
No need to define it. It's already in `AssetBundle` class.

Reply:
done.